Failed to install with npm mentioning a  zip-it-and-ship-it git repo

Over the last 12 hours or so, some Gitlab CI pipelines are failing with a:

```
npm install -g netlify-cli
npm ERR! code ENOENT
npm ERR! syscall spawn git
npm ERR! path git
npm ERR! errno ENOENT
npm ERR! enoent Error while executing:
npm ERR! enoent undefined ls-remote -h -t ssh://git@github.com/netlify/zip-it-and-ship-it.git
npm ERR! enoent 
npm ERR! enoent 
npm ERR! enoent spawn git ENOENT
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent 
```

As this is installed inside a docker container I do not have easy access for further logs (at least, for now)

This has failed while being executed inside a  `node:10.16.3-alpine` (that was running happily for a couple of months now) docker image and a just updated `node:12.13.0-alpine` one.

Does any one have any ideas what causes this?

--- a/package.json
+++ b/package.json
@@ -61,7 +61,7 @@
   },
   "dependencies": {
     "@iarna/toml": "^2.2.3",
-    "@netlify/build": "0.0.26",
+    "@netlify/build": "0.0.27",
     "@netlify/cli-utils": "^1.0.7",
     "@netlify/zip-it-and-ship-it": "^0.3.1",
     "@oclif/command": "^1.5.18",

